IdentityCore/src/auth_scheme/MSIDAuthenticationScheme.h

```diff
@@ -43,7 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) MSIDAuthScheme authScheme;
 @property (nonatomic, readonly) NSDictionary *schemeParameters;
 @property (nonatomic, readonly) MSIDCredentialType credentialType;
-@property (nonatomic, readonly) NSString *tokenType;
+@property (nonatomic, nullable, readonly) NSString *tokenType;
```

Comment:
can ```tokenType``` be nullable?

Reply:
yes, it is nullable for the Bearer type.